JsBasics/Intro_toFunctions/task2function.js

```diff
@@ -0,0 +1,19 @@
+﻿//Завдання 2
+
+//Створіть функцію яка приймає один параметр: age.
+//Усередині функції перевірте, чи age більше або рівне 18.
+//Якщо age відповідає умові, поверніть true, інакше поверніть false.
+//Викличте вашу функцію з аргументами 25 і 15, і виведіть результат(чи є особа дорослою) в консоль для кожного випадку.
+
+function verifyAge(age, over) {
+    if (age >= 18) {
+        console.log("True");
```

Comment:
Тут треба не виводити
```suggestion
        return True;
```